sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/SimplifyConditionalsInPredicateSuite.scala

```diff
@@ -222,7 +225,7 @@ class SimplifyConditionalsInPredicateSuite extends PlanTest {
   }
 
   private def testProjection(originalExpr: Expression, expectedExpr: Expression): Unit = {
-    test((rel, exp) => rel.select(exp), originalExpr, expectedExpr)
+    test((rel, exp) => rel.select(exp), originalExpr.as("out"), expectedExpr.as("out"))
```

Comment:
Add alias to avoid test failure:
```
== FAIL: Plans do not match ===
!Project [CASE WHEN (i#0 > 10) THEN b#0 END AS CASE WHEN (i > 10) THEN b ELSE NULL END#0]   Project [CASE WHEN (i#0 > 10) THEN b#0 END AS CASE WHEN (i > 10) THEN b END#0]
 +- LocalRelation <empty>, [i#0, b#0, a#0, m#0]                                             +- LocalRelation <empty>, [i#0, b#0, a#0, m#0]

```